components/ViewHabits.tsx

```diff
@@ -0,0 +1,22 @@
+import { css } from '@emotion/react';
+import Image from 'next/image';
+
+type Props = {};
+
+const ViewHabits = (props: Props) => {
+  return (
+    <div css={containerStyle}>
+      <Image src='/logo.svg' alt='.SHIFT logo' fill />
+    </div>
+  );
+};
+
+const containerStyle = css({
```

Comment:
[nit] same comment as in the last PR - it'll be more readable to have this above `ViewHabits`, since it is referencing this, yet the variable is declared below. This only really works because the component is a function, but if it was not, it would not be possible to reference a variable that hasn't been declared yet!

Reply:
Interestingly, I spoke to my manager at work about the style of this and especially when you have chains of functions calling each other, is the preference to have the top-level function at the top or bottom of the file? My preference (maybe due to coming from C/C++/Java-y land) was to have the lower level functions defined first, and the functions that call those lower down. My manager said his preference was for the higher level things to be at the top because then there's less need to even scroll down and look at the other functions/variables unless you really need to understand the inner workings of things. I guess I've gotten used to that way of doing things instead, but I'll swap back to my way now?